container-init.d/000-ceramic.sh

```diff
@@ -75,9 +29,6 @@ ipfs repo migrate
 ipfs config Addresses.API "/ip4/0.0.0.0/tcp/$IPFS_API_PORT"
 ipfs config Addresses.Gateway "/ip4/0.0.0.0/tcp/$IPFS_GATEWAY_PORT"
```

Comment:
Should we take this out too?

Reply:
Yes, I was planning on doing that in a separate PR but should probably just do it now.